setup.cfg

```diff
@@ -13,6 +13,7 @@ test = pytest
 
 [tool:pytest]
 testpaths = tests
+pythonpath = data input regrtest_data
```

Comment:
This could be a standalone pull request, it's using pytest config to add fixture directory to the python path.

Reply:
I'm testing this change right now to see if that's all we need to revert the test adjustments in the astroid pr.

Reply:
Doesn't seem to make a difference -- which makes sense based on my limited understanding: the astroid pathways for defining modules are outside the control of pytest, and `tests` definitely is a package, we finally see it now as a namespace package.

Reply:
But please test and see if you get the same result! 🙏🏻

Reply:
(Slightly related we had a lot of issue with no``__init__.py`` in test directory, we could change that if it's causing more problem see https://github.com/PyCQA/pylint/issues/7085#issuecomment-1169863733)

Reply:
I think that's fixed in #7114, which has nothing to do with astroid thankfully, I just figured it out by reading the code comments :D

Reply:
I'm going to test on the 2.12 branch, I don't know why I did not do that earlier.

Reply:
If it helps, I noticed there were subtle behavior differences in running the tests depending on whether you have an editable install. (e.g. whether you're picking up `pylint/tests` or `astroid/tests` as `tests` automatically before you get to namespace package resolution)

Reply:
I removed the commit as I think you're right that astroid is not using the sys.path provided by pytest.